Makefile

```diff
@@ -67,28 +67,27 @@ test-docker-py3:
 
 test-local:
 	# TODO: when we have better coverage, modify this to actually fail when coverage is too low.
-	rm -rf .coverage
-	LUIGI_CONFIG_PATH='config/test.cfg' python -m coverage run --rcfile=./.coveragerc -m nose --with-xunit --xunit-file=unittests.xml -A 'not acceptance'
```

Comment:
Can we create pytest.ini file like this to avoid adding args with every pytest call?
```
[pytest]
addopts = --cov=edx --junit-xml=unittests.xml -s -k
```

Reply:
yes sure, but I think we need `-k` with the expression, so it should not be added as default..